draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1267,7 +1276,7 @@ established with {{-urireg}}.
 
 ### coap+ws

```

Comment:
There are many changes indicated in this section but the diffs look exactly alike?